backend/routes/restaurant.js

```diff
@@ -454,7 +454,7 @@ router.get('/:restaurantID/capacity', (req, res) => {
   }
 
   connection.query(
-    'SELECT MIN(MinGuests) as minimum, MAX(MaxGuests) as maximum FROM restaurant_db.table as t WHERE t.RestaurantID = ?;',
+    'SELECT MIN(MinGuests) as minimum, MAX(MaxGuests) as maximum FROM restaurant_db.TABLE as t WHERE t.RestaurantID = ?;',
```

Comment:
Please merge master to keep your branch up to date (this query no longer should have the 'restaurant_db' reference.

Otherwise good to go!

Reply:
Thanks a lot. Will do it asap